### PR TITLE
Uldaman - OnCreatureGroupDespawn - Prototype

### PR DESCRIPTION
**Pull request by miraco**
This is only an prototype to show and debug OnCreatureGroupDespawn function.

So what is happening here:
We put all scorpions that stay in the room of enchant trainer [Annora](https://tbc.wowhead.com/npc=11073/annora#comments) into an spawn_group.
When all creatures in this spawn_group are dead, the last killed creature will summon Annora.

Video of it:
https://youtu.be/ii4dI9rzrNY

### CODE REVIEW

**Comment by AnonXS** [comment]
https://youtu.be/pb03BDcu-9o?t=1049

**Comment by miraco** [comment]
Tested it on retail.
Can confirm that she gets spawned after all the scorpids in her room are dead. She spawns in the corner and moves to the tent.

I added her waypoints and changed the way she gets spawned with setting her into an spawn group and changing worldstate when scorpid group is dead.

**Comment by killerwife** [comment]
Ok any roadblocks to merging this?

File: sql/updates/mangos/s451_01_mangos_instance_uldaman_prototype.sql [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+-- Put Annora into an spawn_group with worldstate, let her spawn when all scorpids are dead
+-- add waypoints
+UPDATE `creature_template` SET `MovementType`='0' WHERE (`Entry`='11073');
+UPDATE `creature` SET `MovementType`='2' WHERE (`guid`='7000324');
+
+DELETE FROM spawn_group_formation WHERE Id = 7000001;
+INSERT INTO `spawn_group_formation` (`Id`, `PathId`, `MovementType`, `Comment`) VALUES 
+(@SGGUID+1, '11073', '2', 'Uldaman - Annora');
+
+DELETE FROM waypoint_path WHERE PathId = 11073;
```
Comment (AnonXS) [inline sql/updates/mangos/s451_01_mangos_instance_uldaman_prototype.sql:33]: PathId = @SGGUID+1

**Comment by killerwife** [comment]
Closed by https://github.com/cmangos/mangos-tbc/commit/a4394cc31c0ced0c66827483c7ef981d2954b41a
@AnonXS please add working sql

**Pull request closed without merging**